packages/cli/src/lib/implementation/set-verbose.middleware.unit.test.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, expect, it, vi } from 'vitest';
+import {
+  getNormalizedOptionValue,
+  setVerboseMiddleware,
+} from './set-verbose.middleware.js';
+
+describe('setVerboseMiddleware', () => {
+  it.each([
+    [true, true],
+    ['true', true],
+    ['True', true],
+    ['TRUE', true],
+    [false, false],
+    ['false', false],
+    ['False', false],
+    ['FALSE', false],
+    [undefined, undefined],
+    ['undefined', undefined],
+    ['Whatever else', undefined],
+    [0, undefined],
+    [1, undefined],
```

Comment:
Sometimes `0` and `1` are used as environment flags, we could supports those variants as well - `0` would be `false`, any non-zero number would be `true`.